lib/archivers/zip/zip-archive-entry.js

```diff
@@ -26,8 +26,8 @@ var ZipArchiveEntry = module.exports = function(name) {
   this.method = -1;
 
   this.name = null;
-  this.size = -1;
-  this.csize = -1;
+  this.size = 0;
```

Comment:
i think was left over from older code and i thought it got set to 0 from one of the consumer modules but this will ensure such is a sane default. thanks.